.github/workflows/integration_test_run.yaml

```diff
@@ -324,3 +293,17 @@ jobs:
           target: ${{ inputs.zap-target-protocol }}://${{ env.ZAP_TARGET }}:${{ inputs.zap-target-port }}/
           cmd_options: ${{ inputs.zap-cmd-options }}
           rules_file_name: ${{ inputs.zap-rules-file-name }}
+      - name: Pack charm
+        if: always()
+        working-directory: ${{ inputs.working-directory }}
+        run: |
+          # Not ideal, but the charm doesn't get always packed during the integration tests step
+          charmcraft pack
+          echo "CHARM_FILE=$(ls ${{ env.CHARM_NAME }}_*.charm)" >> $GITHUB_ENV
```

Comment:
Perhaps can we check whether it was built and only build it if not?

Reply:
Done. thx